<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bruchpilot</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera view</string>
    </property>
    <widget class="QLabel" name="camview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="camselector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_showtargets">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>420</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show targets</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_showforeground">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>420</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Only foreground</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>661</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="experiment">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Metadata</string>
     </attribute>
     <widget class="QLineEdit" name="md_genotype">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>481</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Genotype:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Fly age:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="md_age">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>days</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Number:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="md_number">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>flies</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Experimenter:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="md_person">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>481</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Sex:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="md_sex">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mixed</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Notes:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="md_notes">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>481</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Project:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selector_project">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stimulus">
     <attribute name="title">
      <string>Experiment</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Protocol set:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selector_protocol_set">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Protocol:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selector_protocol">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cameras">
     <attribute name="title">
      <string>Acquisition</string>
     </attribute>
     <widget class="QGroupBox" name="group_camerafps">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Image acquisition</string>
      </property>
      <widget class="PlotWidget" name="plot_fps_camera">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_recorderfps">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Raw recorder</string>
      </property>
      <widget class="PlotWidget" name="plot_fps_recorder">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_trackerfps">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Target detector</string>
      </property>
      <widget class="PlotWidget" name="plot_fps_tracker">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tracker</string>
     </attribute>
     <widget class="QGroupBox" name="group_numtargets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>2D targets</string>
      </property>
      <widget class="PlotWidget" name="plot_num_targets">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_record">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>640</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
background-color: rgb(255, 0, 0);
}
QPushButton {
background-color: rgb(0, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Start experiment</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_snapshot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Snapshot</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Recording options</string>
    </property>
    <widget class="QCheckBox" name="check_recordraw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Store raw video</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_snapshot_repeat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat snapshot</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
